omaha/base/constants.h

```diff
@@ -281,6 +281,14 @@ const TCHAR* const kRegValueAutoUpdateJitterMs      = _T("AutoUpdateJitterMs");
 const TCHAR* const kRegValueProxyHost               = _T("ProxyHost");
 const TCHAR* const kRegValueProxyPort               = _T("ProxyPort");
 const TCHAR* const kRegValueMID                     = _T("mid");
+const TCHAR* const kRegValueAlwaysVerifyAuthenticodeSignatures =
+    _T("AlwaysVerifyAuthenticodeSignatures");
```

Comment:
I suggest the naming be more specific in this case, for example VerifyPayloadAuthenticodeSignature or something like that. Doing this gets right to the point of feature, which is is verifying the downloads, there is no uncertainty about what is being verified. Authenticode occurs in several contexts and it's useful to know precisely what is being enforced.

Reply:
Fixed in 9477eacac348e46b2d643ef7e2e843a48947293c.